app/serialised_models.py

```diff
@@ -92,8 +92,8 @@ def get_dict(template_id, service_id, version):
         return {"data": template_dict}
 
     @cached_property
-    def email_files(self):
-        return SerialisedTemplateEmailFileCollection.from_template_id(self.id, self.version)
+    def email_file_objects(self):
```

Comment:
do we even need this if we already have the email files available on the template?

Reply:
I mean, do we even need `SerialisedTemplateEmailFile` and `SerialisedTemplateEmailFileCollection`?